File: src/main/java/nuclearcoder/discordbot/command/misc/CmdUwu.java
package nuclearcoder.discordbot.command.misc;

import nuclearcoder.discordbot.NuclearBot;
import nuclearcoder.discordbot.command.Command;
import sx.blah.discord.handle.obj.IMessage;

public class CmdUwu implements Command {

    private static final String UWU = "https://puu.sh/sAQuH/29cca0f3cd.jpg";

    @Override public void execute(NuclearBot bot, IMessage message, String command, String[] args)
    {
        bot.sendFileURL(message.getChannel(), "", UWU, "uwu.jpg");
    }

}

File: src/main/java/nukebot/command/misc/CmdUwu.kt
package nukebot.command.misc

import nukebot.NuclearBot
import nukebot.command.Command
import nukebot.rSendFileURL
import sx.blah.discord.handle.obj.IMessage

class CmdUwu : Command {

    override fun execute(bot: NuclearBot, message: IMessage, command: String, args: Array<String>) {
        message.channel.rSendFileURL("", "https://puu.sh/sAQuH/29cca0f3cd.jpg", "uwu.jpg")
    }

}
